CMakeLists.txt

```diff
@@ -24,25 +24,35 @@ if (UNIX AND NOT APPLE AND NOT ANDROID AND NOT WEBGL)
 endif()
 
 if (WIN32)
+    option(USE_STATIC_CRT "Link against the static runtime libraries." ON)
+
     # On Windows we need to instruct cmake to generate the .def in order to get the .lib required
     # when linking against dlls. CL.EXE will not generate .lib without .def file (or without pragma
     # __declspec(dllexport) in front of each functions).
     set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
 
+    if (${USE_STATIC_CRT})
+        set(_CRT_FLAGS_RELEASE "/MT")
+        set(_CRT_FLAGS_DEBUG "/MTd")
+    else()
+        set(_CRT_FLAGS_RELEASE "/MD")
+        set(_CRT_FLAGS_DEBUG "/MDd")
+    endif()
+
     # TODO: Figure out why pdb generation messes with incremental compilaton.
     # IN RELEASE_WITH_DEBUG_INFO, generate debug info in .obj, no in pdb.
-    set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "${CMAKE_CXX_FLAGS_RELWITHDEBINFO} /Z7")
-    set(CMAKE_C_FLAGS_RELWITHDEBINFO "${CMAKE_C_FLAGS_RELWITHDEBINFO} /Z7")
+    set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "${CMAKE_CXX_FLAGS_RELWITHDEBINFO} ${_CRT_FLAGS_RELEASE} /Z7")
+    set(CMAKE_C_FLAGS_RELWITHDEBINFO "${CMAKE_C_FLAGS_RELWITHDEBINFO} ${_CRT_FLAGS_RELEASE} /Z7")
 
     # In RELEASE, link statically against c/c++ lib to avoid missing redistriburable such as
```

Comment:
You should update this comment, it's not entirely correct after your change

Reply:
Ah yes, missed this one and updated it.